Content.Shared/Camera/CameraRecoilComponent.cs

```diff
@@ -13,5 +14,6 @@ public sealed partial class CameraRecoilComponent : Component
     /// <summary>
     ///     Basically I needed a way to chain this effect for the attack lunge animation. Sorry!
     /// </summary>
+    [ViewVariables, AutoNetworkedField]
```

Comment:
This isn't used anywhere upstream I think, but if this is needed it should be made as a separate PR upstream. Same with the AutoGenerateComponentState attribute.

Reply:
Upstream changes reverted